sql/src/main/java/io/crate/expression/reference/file/LineContext.java

```diff
@@ -66,8 +58,27 @@ public Object get(ColumnIdent columnIdent) {
         return val;
     }
 
+    public Map<String, Object> sourceAsMap() {
+        if (parsedSource == null) {
+            try {
+                parsedSource = XContentHelper.convertToMap(new BytesArray(rawSource), false, XContentType.JSON).v2();
+            } catch (NullPointerException e) {
+                return null;
```

Comment:
Note: Doesn't look correct that we return null here (but was like this before).

Reply:
You changed the order of the methods. We try to prevent unnecessary changes because it makes reviewing and tracking the Git changelog harder.

Reply:
Sorry about that - usually would make those changes in a prefactor/refactor PR!